packages/sol-cov/src/revert_trace_subprovider.ts

```diff
@@ -88,16 +89,19 @@ export class RevertTraceSubprovider extends TraceCollectionSubprovider {
             // actually happens in assembly). In that case, we want to keep
             // searching backwards by decrementing the pc until we find a
             // mapped source range.
-            while (_.isUndefined(sourceRange)) {
+            while (_.isUndefined(sourceRange) && pc > 0) {
                 sourceRange = pcToSourceRange[pc];
                 pc -= 1;
-                if (pc <= 0) {
-                    this._logger.warn(
-                        `could not find matching sourceRange for structLog: ${evmCallStackEntry.structLog}`,
-                    );
-                    continue;
```

Comment:
What I meant to do here was continue to the next iteration of the `for` loop, but since we're inside a `while` loop it was just continuing to the next iteration of the `while` loop which doesn't really do anything. If the `sourceRange` was undefined, this caused an infinite loop.